packages/sandcastle/templates/Sandcastle.ts

```diff
@@ -114,20 +114,27 @@ const Sandcastle = {
     toolbarId?: string,
   ) {
     Sandcastle.declare(onchange);
+
     const input = document.createElement("input");
     input.checked = checked;
     input.type = "checkbox";
     input.style.pointerEvents = "none";
+    input.className = "stratakit-mimic-switch";
+
     const label = document.createElement("label");
-    label.appendChild(input);
     label.appendChild(document.createTextNode(text));
     label.style.pointerEvents = "none";
-    const button = document.createElement("button");
-    button.type = "button";
-    button.className = "cesium-button";
-    button.appendChild(label);
+    label.className = "stratakit-mimic-label";
 
-    button.onclick = function () {
+    const field = document.createElement("div");
+    // button.type = "button";
+    field.className = "stratakit-mimic-field";
+    field.dataset.kiwiLabelPlacement = "after";
```

Comment:
"kiwi" is non-obvious to those who don't know the previous name of stratakit. I think that warrant a comment to explain.